challenge-sizebay/src/Contexts/GlobalState.jsx

```diff
@@ -0,0 +1,24 @@
+import React, {useState} from 'react'
+import {GlobalContext} from './GlobalContext'
+
+const GlobalState = (props) => {
+const [pending, setPending] = useState(['Tarefa pendente'])
+const [done, setDone] = useState([])
+const [listToShow, setListToShow] = useState('') 
+const [search, setSearch] = useState('')
+const [taskProgress, setTaskProgress] = useState([])
+const [percentagesArray, setPercentagesArray] = useState([])
+
+    
+const states = {pending, done, listToShow, search,taskProgress, percentagesArray }
+const setters = {setPending, setDone, setListToShow, setSearch, setTaskProgress, setPercentagesArray}
+const data = {states, setters}
```

Comment:
nit: brackets should have a space between their content.

Reply:
Done!